### DISCUSSION TITLE
tox-uv how to see what versions are installed in the tox env?

### DISCUSSION THREAD

**Discussion by anilbey** [tox-dev/tox#3231] [Q&A] [answered]
Hi with `tox-uv` I have the following files under `/.tox/py3.venv`.
How can I see what versions of packages are installed in there?

```

❯ tree -L 1                                                                                                                                                                                 ─╯
.
├── bin
├── CACHEDIR.TAG
├── lib
├── pyvenv.cfg
└── share

3 directories, 2 files
```

In normal `tox`, I could run commands like `bin/pip list`. What is the equivalent of it in `uv`?

**Comment by gaborbernat** [comment, marked as answer]
You can use `uv` for that, the virtual environment is inside the `.venv` folder inside the tox folder, so just need to point uv at that.
```bash
env VIRTUAL_ENV=.tox/py3.venv/.venv uv pip list
```
